### DISCUSSION TITLE
histogram based xgboost example

### DISCUSSION THREAD

**Discussion by abbasabdirashid** [NVIDIA/NVFlare#1715] [Q&A] [answered]
hi i ran the histogram-based xgboost example with the simulator and i was wondering if there is a way to see which information are the clients sending to the server (histograms in this case)?

**Comment by abbasabdirashid** [comment]
* and vice versa

**Comment by YuanTingHsieh** [comment]
@ZiyueXu77 or @rongou can you help to answer this question? thanks in advance.

**Reply by rongou** [reply, marked as answer]
We don't have logging specifically for the federated communication. You might have to add some logging statements to https://github.com/dmlc/xgboost/blob/master/plugin/federated/federated_client.h to see what's being communicated.
